esp/bindings/SOAP/xpp/fxpp/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+################################################################################
```

Comment:
Does an empty CMakeLists.txt file serve a purpose?

Reply:
I can't think of one. I just followed the lead of the xpp, xpp/xpp, and xpp/sxt directories, which also include empty files. I figured it gave me a 75% chance of being right, since xpp/xjx is the only parser directory that doesn't have an empty file.